homeassistant/components/obihai/sensor.py

```diff
@@ -0,0 +1,99 @@
+"""Support for Obihai Sensors."""
+import logging
+
+from datetime import timedelta
+import voluptuous as vol
+
+from pyobihai import PyObihai
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import STATE_UNKNOWN
+from homeassistant.const import CONF_HOST, CONF_PASSWORD, CONF_USERNAME
+
+from homeassistant.helpers.entity import Entity
+import homeassistant.helpers.config_validation as cv
+
+
+_LOGGER = logging.getLogger(__name__)
+
+SCAN_INTERVAL = timedelta(seconds=5)
+
+DOMAIN = "Obihai"
```

Comment:
We don't define the domain constant for platforms. Please use another constant variable name.